packages/framework/collabspaces/src/test/collabSpaces.spec.ts

```diff
@@ -309,6 +310,104 @@ describe("Temporal Collab Spaces", () => {
 		assert(value2?.value === initialValue, "value was preserved correctly");
 	});
 
+	describe("Reverse Mapping tests", () => {
+		async function getRowAndColIdsFromChannelId(
+			collabSpace: IEfficientMatrix & IEfficientMatrixTest,
+			row: number,
+			col: number,
+		): Promise<{ rowId: string; colId: string }> {
+			const result = await collabSpace.getCellChannelIdDebugInfo(row, col);
+			const parts = result.channelId.split(",");
+			assert(parts.length === 3, "Invalid channel Id");
+			const rowId = parts[0];
+			const colId = parts[1];
+			return { rowId, colId };
+		}
+
+		it("Reverse Mapping: Basic test", async () => {
+			const collabSpace = await initialize();
+			const row = 5;
+			const col = 3;
+			const { rowId, colId } = await getRowAndColIdsFromChannelId(collabSpace, row, col);
+
+			const { rowMapSize, colMapSize, rowIndex, colIndex } =
+				collabSpace.getReverseMapsDebugInfo(rowId, colId);
+
+			assert(rowMapSize === rows, "rowMapSize is incorrect");
+			assert(rowIndex === row + 1, "rowIndex from the actual matrix has to be offset by 1");
+			assert(colMapSize === cols, "colMapSize is incorrect");
+			assert(colIndex === col + 1, "colIndex is correct");
```

Comment:
I'd suggest for API surface to be consistent. I.e. for getReverseMapsDebugInfo() to do -1, and thus all the code (including test code) not to deal with +1 or -1. We could change (in the future) where and how we store this data, it would be great (even for test code) not to depend on a fact that we have an extra row / col